hypervisor/virtualbox/virtualbox.go

```diff
@@ -51,8 +50,11 @@ func (*virtualbox) Create(imageName, vmName string, cpus int, mem int) error {
 		"--memory", strconv.Itoa(mem),
 	}
 
+	// Wrap each argument in quotes if it contains spaces
 	for i, arg := range importArgs {
-		importArgs[i] = `"` + arg + `"`
+		if strings.Contains(arg, " ") {
+			importArgs[i] = `"` + arg + `"`
+		}
```

Comment:
do we need to revert the old code (related to CPU and memory flags)?

![image](https://github.com/mhewedy/vermin/assets/1086049/a6955ec6-6f48-4435-a8fb-6181eb1bde04)

Reply:
I suggest to revert the old code, and then just fix the parts that contains the filesystem path

Reply:
I'm getting error 

_VBoxManage: error: Invalid command '"import"'_

I guess mac is not able to take the import argument in string format like windows

cpus and memory flags are not the issue

Reply:
@mhewedy reverted to old style of code just updating the paths to include quotes


now tested on windows, mac and linux

Reply:
Thanks